Cloud_Community_REST_API_Docs.md

```
@@ -0,0 +1,1060 @@
+}
+```
+
+---
+
+### 특정 회원 조회
+
+**GET** `/users/{nickname)`
+
+다른 사용자의 공개 프로필을 조회합니다. 비밀번호는 제공하지 않습니다.
+
+**Path Parameters**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)